paragraphs_jquery_ui_accordion.module

```diff
@@ -133,6 +133,9 @@ function paragraphs_jquery_ui_accordion_field_formatter_view($entity_type, $enti
   $paragraphs_items = entity_load('paragraphs_item', $paragraphs_ids);
 
   foreach ($paragraphs_items as $paragraphs_item) {
+    if ($paragraphs_item->status != 1) {
```

Comment:
Where the `status` property of a `paragraph_item` object is used in the `paragraphs` module, it treats it as a boolean (which it is) rather than comparing with a numerical value.

We should do the same here.

```suggestion
    if (!$paragraphs_item->status) {
```

Reply:
@yorkshire-pudding  I have updated the PR.
https://github.com/backdrop-contrib/paragraphs_jquery_ui_accordion/pull/19/commits/49eba16e0d7eaf534de6f198f54c7b287ad69d5c
Please Check.